### PR TITLE
Handle 401 & 403 Server Responses Correctly

### PR DESCRIPTION
**Pull request by mark-monteiro**
* Update the login page to handle a 403 response from the server. A new message/translation string has been added to to display to the user in this case.
* Update to expect a 403 response for parental control limitations

The associated server updates are in https://github.com/jellyfin/jellyfin/pull/2861

### CODE REVIEW

File: src/controllers/auth/login.js [hunk 1]
```diff
@@ -24,9 +24,10 @@ define(["apphost", "appSettings", "dom", "connectionManager", "loading", "layout
             page.querySelector("#txtManualPassword").value = "";
             loading.hide();
 
-            if (response.status === 401) {
+            if (response.status === 401 || response.status === 403) {
```
Comment (MrTimscampi) [inline src/controllers/auth/login.js:27]:
```suggestion
            const UnauthorizedOrForbidden = [401, 403]
            if (UnauthorizedOrForbidden.inclues(response.status)) {
```

It doesn't make it shorter, but it's a bit more obvious and we can more the error codes to some utility module afterward.

For reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

File: src/controllers/auth/login.js [hunk 2]
```diff
@@ -24,9 +24,10 @@ define(["apphost", "appSettings", "dom", "connectionManager", "loading", "layout
...
                 require(["toast"], function (toast) {
-                    toast(Globalize.translate("MessageInvalidUser"));
+                    var messageKey = response.status === 401 ? "MessageInvalidUser" : "MessageUnauthorizedUser";
```
Comment (MrTimscampi) [inline src/controllers/auth/login.js:29]: Feel free to use `let` or `const` in modules, everything is passed through Babel (not only what is explicitly set to be an ES6 module).

File: src/controllers/auth/login.js (hunk 1 above)
Comment (mark-monteiro) [reply src/controllers/auth/login.js]: Done 👍

File: src/controllers/auth/login.js (hunk 2 above)
Comment (mark-monteiro) [reply src/controllers/auth/login.js]: Done 👍 So nice to be able to use modern features :D

Comment (MrTimscampi) [reply src/controllers/auth/login.js]: We have polyfill linting too, now, so you can use new APIs and it'll warn you if stuff needs a polyfill :)

File: src/controllers/auth/login.js (hunk 1 above)
Comment (dkanada) [reply src/controllers/auth/login.js]: I think the other way was more readable for what it's worth.

**Comment by MrTimscampi** [comment]
Requires jellyfin/jellyfin#2861 to be merged first.

**Pull request merged**